src/Renderer.js

```diff
@@ -35,12 +36,16 @@ export function Renderer(options) {
     this.process();
   });
 
-  // we expect to load a renderer url once only so cache the request to load script
-  loadScript(url, this.callback, true);
+  if (!isRendererDefinedOnAdUnit(adUnitCode)) {
+    // we expect to load a renderer url once only so cache the request to load script
+    loadScript(url, this.callback, true);
+  } else {
+    utils.logWarn(`External Js not loaded by Renderer since renderer url and callback is already defined on adUnit ${adUnitCode}`);
```

Comment:
Does this need to be a warning?  

Given the nature of the setup (ie defining a renderer in the adunit), this seems like a `logInfo` type message since the user isn't doing anything wrong and the code is handling this scenario as expected.  Unless I'm missing something?

Reply:
Here is the user has set that renderer defined on adUnit will render the bid but the demand partners they are working with are using different js to render.